roles/ceph-osd/tasks/scenarios/journal_collocation.yml

```diff
@@ -12,7 +12,7 @@
 # failed, this is why we check if the device is a partition too.
 - name: automatic prepare osd disk(s) without partitions
   command: ceph-disk prepare "/dev/{{ item.key }}"
-  ignore_errors: true
+  failed_when: false
```

Comment:
I'd like to keep this one with `ignore_errors: true`. I don't want to have hidden.
It can be useful to see which disk didn't succeed to get formatted or something else.